.github/CODEOWNERS

```diff
@@ -27,6 +27,9 @@
 # all Boundary resources and clients are owned by the boundary-cloud team
 *boundary* @hashicorp/boundary-cloud
 
+# all Waypoint resources and clients are owned by the waypoint team
+**/waypoint @hashicorp/cloud-waypoint
+
 # Statuspage.io configuration is owned by the Core SRE team
 /internal/provider/statuspage.go @hashicorp/core-sre
 /internal/providersdkv2/statuspage.go @hashicorp/core-sre
```

Comment:
Non blocking comment: Github claims that the core-sre team is not a valid team to own these two files and so it says that there are errors on line 34 and 35 but you didn't touch those lines so I assume the issue is pre-existing or just github making a mistake.

Reply:
I assume this is pre-existing as well. I'm not sure why.

Reply:
Core-SRE team has dissolved. We need to find new owner. Kindly ignore for now.